### ISSUE TITLE
Gutenberg Rapid Website Recreation Videos - Feedback wanted 😀

### ISSUE THREAD

**Issue by jamiemarsland** [WordPress/Learn#1439]
Hi Everyone, 

It would be great to get some feedback on this and whether you think this would fit into the learn wordpress catalog somewhere. 

Rapid Website Recreation Video

Following a discussion with Jonathan Bossenger here's an example video from my youtube channel.  This is one of a series of videos where I try to show what's possible with Gutenberg and Block Themes. 

https://youtu.be/crsb1HJYJgk

They generally get great feedback and engagement and they are a good way of inspiring people to give block themes and gutenberg a go. 

Here is a playlist of other videos that follow the same format https://youtube.com/playlist?list=PLiPorWuYVbzuVrydFJZflALOc5rpUuwgs

In most cases I'm using core gutenberg with the twenty twenty three theme

**Comment by kaitohm** [comment]
Just pinging you @jonathanbossenger seeing you were mentioned in this issue.

**Comment by jonathanbossenger** [comment]
Thanks for the ping @bsanevans, and thanks for creating the issue @jamiemarsland 

I had a busy week last week, so I intend to do a review this week.

**Comment by ghost** [comment]
Definitely think this is something people are looking for. The video is short so people are willing to take the time to watch. Simple answers to questions. Step by step instructions. Query loop is a topic that I have seen a lot of interest in. I love your circle cursor and how it magnifies things. Do you have a video on how to set that up?

**Comment by jonathanbossenger** [comment]
@jamiemarsland  I just watched this and am excited about including this type of content on Learn WordPress. 

You bring the perspective of someone using this in the wild, which is exactly what our learners need. If you're available, I'd encourage you to consider hosting a few Learn WordPress [online workshops](https://make.wordpress.org/training/handbook/online-workshops/) as well. 

- I love how you're using core WordPress and a default theme to build this
- I also appreciate how you're using any of the free and open-source plugins available in the WordPress repository to add functionality that's not doing exactly what you need in core WordPress. 
- Last but not least, showing how you're using things like the list view, and/or shortcuts helps to showcase valuable tips and tricks for site developers! Great stuff.

My only comment, and it's going to be about the reason we started chatting in the first place, is the section from around 13:52. 

It's not the cat (🙂) and more about the fact that that section asking folks to "click the like/subscribe button" is very specific to watching the video on YouTube. If this video was uploaded to WordPress.tv and then hosted as a tutorial on Learn WordPress, that would be confusing. 

So my only suggestion here would be if it would be possible to have something more along the lines of what's described in the [section in the promotional guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/#make-reasonable-follower-callouts) ie "subscribe to my channel for more updates". Personally, I have no problem with you saying something like "if you subscribe for updates my cat gets a treat", and the associated cat video. 

Although I do worry you mind end up becoming "the cat treat guy". 🤣 But perhaps you're ok with that? 😊

**Comment by jonathanbossenger** [comment]
Note, even though this is actually a finished tutorial, I'm moving it to drafts in progress for now, as I've suggested a minor update. Once we all agree it's ready to go, we can accept the two current reviews as part of the review process, and label accordingly.

**Comment by jamiemarsland** [comment]
Thanks for the fab feedback @jonathanbossenger  - 100% on board with all that 👍
fyi today I ran my first 'hands-on' master site editing and block theme course (through my pootlepress business) . Would love to share my feedback - i've been running hands on wp beginner courses for 10 years, and there's nothing quite like a hands on course to see what people find difficult :)

**Comment by jamiemarsland** [comment]
Here's the edited video :) https://vimeo.com/816574139

**Comment by nomad-skateboarding-dev** [comment]
I agree with @jonathanbossenger and others, the content here is great, and @jamiemarsland is a fantastic presenter. Informative and enjoyable to watch!

My feedback/concern has to do with connecting Learn content to monetized content in general.

All of the Pootlepress how-to videos are ads for Pootlepress products, which I feel obfuscates the line between what WordPress can do out of the box, and what is being demonstrated in the how-to videos.

These instant websites being an example of what the sell is.

![Screenshot_20230411-152107.png](https://user-images.githubusercontent.com/78917704/231302107-92446938-b8be-4cda-aa69-9b70c0b7247f.png)

You're purchasing default WordPress with less content than the default theme patterns provide, with the selling points being core WordPress inclusions: mobile friendly, editable, has a gallery, etc.

I would include much of the current content on the YouTube channel with this. It both explains how to use WordPress, but with a tone of "it's not super easy, unless you purchase what I'm selling, then you're all set".

To be clear, I'm not opposed to marketing and growing a business. And as I began, Jamie, I think the content you produce is excellent.

But I do have a lot of reservations of putting the official stamp of .org on a business that's monetizing the Learn content, especially when there's hundreds of people making similar content that's not monetized, but given out to the community in the same spirit we are here.

As always, I'm supportive of whoever and whatever supports WordPress and democratizing publishing. So whatever works best for everyone, I will be all in with. 🙂

But I can't emphasize enough how strongly I feel about blurring the lines here, especially when the major critique for the WordPress project since inception has been that the .com side is in control and pushing their own agendas to monetize.

Edit: typo

**Comment by jamiemarsland** [comment]
Hi, thanks for the feedback - a few comments. 

1) There's no upsell at all in the video - the instant websites idea came about after i'd actually shot these rapid recreation videos. As a piece of video content they are just meant to inspire folks to give gutenberg and site editing a go. I try as much as possible to do with just core. I'm one of the only youtubers out that that follow this philisophy. Which is partly why the videos have been sucessful. 

2) The vast majority of my videos have absolutely no upsell to pootlepress products - check out my gutenberg playlist - i've now created over 150 videos on gutenberg that are completely unrelated to pooltepress https://youtube.com/playlist?list=PLiPorWuYVbztdvJxv3ZEbdnPYricJ1XwY

**Comment by nomad-skateboarding-dev** [comment]
I appreciate your reply, Jamie. 

My view may be off, but I am sharing it from an informed perspective. I've seen all of your videos, I follow Pootlepress, and I receive your newsletter.

I would respectfully disagree that there is no upsell. That is exactly why I felt so strongly here, I have never seen a piece of content you've put out that didn't have an upsell. 

As I stated in my comment above, I also feel the upsell is not in the customers best interest, to put it in the best light I can.

The links you provided to your videos all have upsells in them. 

![Screenshot_20230411-221732.png](https://user-images.githubusercontent.com/78917704/231358210-930cf4d9-0021-4d6c-8b28-ee0e7bdd9f71.png)

![Screenshot_20230411-221722.png](https://user-images.githubusercontent.com/78917704/231358307-5184c045-a289-44c3-b4ef-95aacac1a00b.png)

![Screenshot_20230411-222020.png](https://user-images.githubusercontent.com/78917704/231358865-c80e936a-52b0-4891-8adb-63fcc4f4995e.png)

**Comment by jamiemarsland** [comment]
But there's NO upsell whatsoever in the video? 

Please let me know if i've missed something. 

Please could you focus on the video, as this is all we are proposing to show. 

It's just the video that we are proposing to show (no links, no upsells, certainly not my newsletter????)

The links in the youtube descriptions won't appear.

This isn't relevant to this discussion, but the vast majority of my youtube videos have absolutely no upsells in the videos. I have standard links in the description, but again these will not be shown. 

Thanks for any clarification - but please try and focus on the content in the actually video, as this is all that will be shown.

**Comment by jonathanbossenger** [comment]
Hey, @nomad-skateboarding-dev thanks for your review and feedback.

Could I confirm something, does your feedback pertain to the specific video that @jamiemarsland has shared [here for review](https://vimeo.com/816574139), or all of Jamie's other online content?

For some context, the training team has discussed with Jamie the fact that any video content he submits to Learn WordPress has to be uploaded to WordPress.tv, as well as follow both the [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) and the [Promotional Guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/). 

Would you agree that in this specific video, he is adhering to both?

**Comment by nomad-skateboarding-dev** [comment]
I appreciate your reply again @jamiemarsland . I don't appreciate being told where my focus needs to be, as I feel it's already here.

@jonathanbossenger I'm specifically talking about this video. Right at the 14-minute mark, it says go see me on YouTube and go check out all my videos.

When you follow any link to any content Jamie has, there's an upsell for people who are ignorant on WordPress. They believe, for example, that spending $50 gives them the ability to edit the entire website, have it mobile optimized, or have a gallery. When the opposite is true.

To reiterate and be clear: I have no problem with selling things and people making money. I love to see WordPress people and businesses thrive and succeed. 

I'm very strongly opposed to selling things based on people's ignorance.

Very easy way to see my thoughts: purchase an instant website and see what you receive.

Edit: typo

**Comment by jamiemarsland** [comment]
@jonathanbossenger my understanding was that having a call out to youtube/twitter etc was within the guidelines - if not let me know and I'll happily re-edit :)

**Comment by westnz** [comment]
Hey Jamie! I love your rapid website recreation videos. None of our [video tutorials](https://learn.wordpress.org/tutorials/) on Learn has callouts to youtube/twitter etc. I would suggest re-editing 😃  We usually tell folks they can go to learn.wordpress.org for more video tutorials and training material.

**Comment by jonathanbossenger** [comment]
@westnz I don't think it's fair to say none of our video tutorials on Learn has callouts to YouTube or Twitter etc. For example, this [block development tutorial](https://learn.wordpress.org/tutorial/editor-de-bloques-puesta-en-marcha-de-un-entorno-de-desarrollo/) includes the speaker's Twitter handle in the slides, as well as in the credits of the tutorial. I do agree that not many tutorial creators have included such callouts, but it does happen, and it is allowed according to our guidelines.

Rather than re-edit at this point, I would prefer it if we took this discussion to the training team, perhaps in a blog post, or a future meeting, and see what the rest of the team thinks.

My concern is that if we remove the "[reasonable follower callouts](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/#make-reasonable-follower-callouts)" section of the guidelines, which in my opinion is only fair, we remove some of the benefits for contributors to create workshops or tutorials for Learn WordPress.

**Comment by kaitohm** [comment]
Hi all. There's been a bit discussed here, so let me split my comments out.

---

>It would be great to get some feedback on this and whether you think this would fit into the learn wordpress catalog somewhere.
>
>Rapid Website Recreation Video

Yes, I believe rapid website recreation videos have a place on Learn WordPress. I'm currently running a multi-part Online Workshop on a similar topic - #1306 In these series, I've taken my own personal blog and am changing the theme on a local installation. I'm changing themes, but recreating the current look in the Block Editor. I'm doing this while taking live Q&A from the audience.

I'm considering what content type would be most appropriate for what you've suggested. Currently, Tutorials on Learn WordPress have [a defined learning objective](https://make.wordpress.org/training/handbook/lesson-plans/how-to-finalize-description-and-objectives/). (Creators are asked to provide this when starting the creation process on GitHub.) They help viewers achieve a specific task, or gain knowledge regarding a specific WordPress feature. I'm not sure what the learning objective would be in the [video that was shared here](https://vimeo.com/816574139).

If we can define a learning objective for these videos, then I can see them becoming Tutorials. But while I don't have a well-defined learning objective right now, I'm thinking Online Workshops would be more suited for this content. Online Workshops welcome sharing ideas, exploring issues, and networking over shared interests.

To use this content as an Online Workshop, we could either:
* Run similar recreation activities live in Online Workshops, or
* Play the video provided, possibly pausing throughout for questions, or opening the floor for Q&A afterwards

---

>my understanding was that having a call out to youtube/twitter etc was within the guidelines

This is currently accepted in the Training Team handbook. I myself link to my Slack and Twitter handles in all my Online Workshop slides, encouraging folks to come and ask me additional questions there if we run out of time.

Currently, you say the following at the end of your video:

> Thank you so much for watching. I hope you found that useful.
> If you want to see more, I've done over a hundred videos just on Gutenberg over on YouTube.
> You can find me there at `@jamiewp`, or on Twitter at `pootlepress`, or at `pootlepress.com`.
> Thanks again for watching. Bye for now.

The team's [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) say:

> You may not highlight your business or otherwise advertise your own products/services as part of any content on Learn WordPress – this includes individuals who create training content professionally.

Checking the mentioned accounts, YouTube and Twitter seem like personal accounts to me, whereas `pootlepress.com` seems like a business website directly selling products. I think that URL should be removed.

Also, when published, people view the speaker as a representative of the WordPress project. To that end, I think content on Learn WordPress should be predominantly promoted over personally created content.

What are folks' thoughts to rephrasing the ending of the video as follows?

> Thank you so much for watching. I hope you found that useful.
> If you want to see more, check out the other great content on learn.wordpress.org.
> You can also find me on YouTube at `@jamiewp`, or on Twitter at `pootlepress`.
> Thanks again for watching. Bye for now.

---

> My feedback/concern has to do with connecting Learn content to monetized content in general.

This is an important point to consider, and I'm glad it's come up.

Currently, all Tutorial and Online Workshop facilitators must be vetted before they're approved to create content for Learn WordPress. In summary, [the vetting criteria](https://make.wordpress.org/training/handbook/tutorials/reviewing-tutorial-presenter-applications/) is that:

* They must embrace the WordPress license. (If they are distributing WordPress-derivative works (themes, plugins, etc.), they must be 100% GPL compatible.)
* They must respect the WordPress trademarks.

While not listed in "eligibility of presenters", here are additional points listed under "[vetting the applicant](https://make.wordpress.org/training/handbook/tutorials/reviewing-tutorial-presenter-applications/#2-vet-the-applicant)".

* They have not indicated bigoted or discriminatory behavior.
* They have no record of being unreliable.

In short, the team currently looks at licensing and trademark usage by the applicant, that the applicant is equitable to all people, and that they're reliable. As long as businesses and products are not promoted in the content itself, then the team hasn't checked whether businesses/products are mentioned in the social accounts shared in the content.

---

>Rather than re-edit at this point, I would prefer it if we took this discussion to the training team, perhaps in a blog post, or a future meeting, and see what the rest of the team thinks.

Let me know if my comments above have helped draw us to a consensus at all. If they have, then great! If they haven't, then that's ok, too. I can work with someone to draft a post for us to share with the wider team to get more feedback on.

**Comment by jamiemarsland** [comment]
Hi @bsanevans could you clarify this para please?

"As long as businesses and products are not promoted in the content itself, then the team hasn't checked whether businesses/products are mentioned in the social accounts shared in the content."

Oh and also I'm 100% fine with having no call out whatsoever - it seems to be causing a huge amount of discussion and energy, when my aim is just to help inspire people to use gutenberg and wordpress.

**Comment by westnz** [comment]
>  I would prefer it if we took this discussion to the training team

>  I can work with someone to draft a post for us to share with the wider team to get more feedback on.

Sounds like a plan

> my aim is just to help inspire people to use gutenberg and Wordpress.

Absolutely.  These are just some of the discussions that happen in an open-source community. Thanks for your patience Jamie!

**Comment by kaitohm** [comment]
> Hi @bsanevans could you clarify this para please?
>
>"As long as businesses and products are not promoted in the content itself, then the team hasn't checked whether businesses/products are mentioned in the social accounts shared in the content."

Sure. So [in a comment above](https://github.com/WordPress/Learn/issues/1439#issuecomment-1504659761), it was brought up how videos on your YouTube channel have "upsells" in their descriptions. If I'm reading the discussion accurately, the concern is that:

1. [Your proposed video](https://vimeo.com/816574139) doesn't include upsells, just a comment to your YouTube channel
2. But some videos on that YouTube channel do include upsells to products
3. And therefore, if uploaded as-is, there would be an indirect link between a piece of content on Learn WordPress, and monetized content elsewhere

While the above would be true, to the best of my knowledge, the Training Team has not historically done what I have labeled as step 2. Until now, the Training Team's process has been:

1. If the video includes no promotion of a business or product, and only briefly invites people to subscribe to their social network channel/account,
2. Then, the video can be uploaded

We have not checked whether social media accounts mentioned in the video then promote monetized content beyond that. All we have checked is whether businesses or products are mentioned in the video itself. And if not, then we have published these videos.

So, from my perspective, with a minor tweak to the ending (removing the business's URL), the video you edited could be used on Learn, meeting our handbook requirements and following how the team has historically approved content.

---

Separately, if some team members feel our current process could tie content on Learn to monetized content elsewhere too easily, or they recommend additional checks should be introduced when vetting content/speakers, then that is an important conversation worth publishing on the team blog for the wider community to weight in on. I'd be happy to help author that post.

**Comment by jamiemarsland** [comment]
Thanks for the clarification @bsanevans  👍 Can I double check one thing - it seems a call out to a commercial company is ok in the presenter bio from what I can see on https://learn.wordpress.org/ ?

**Comment by nomad-skateboarding-dev** [comment]
Thank you to everyone here for having this conversation. Thank you especially and sincerely @jamiemarsland . 

> Sure. So [in a comment above](https://github.com/WordPress/Learn/issues/1439#issuecomment-1504659761), it was brought up how videos on your YouTube channel have "upsells" in their descriptions. If I'm reading the discussion accurately, the concern is that:
> 
> 1. [Your proposed video](https://vimeo.com/816574139) doesn't include upsells, just a comment to your YouTube channel
> 2. But some videos on that YouTube channel do include upsells to products
> 3. And therefore, if uploaded as-is, there would be an indirect link between a piece of content on Learn WordPress, and monetized content elsewhere

That is very succinct, and exactly sums up my thoughts above @bsanevans 

> Separately, if some team members feel our current process could tie content on Learn to monetized content elsewhere too easily, or they recommend additional checks should be introduced when vetting content/speakers, then that is an important conversation worth publishing on the team blog for the wider community to weight in on. I'd be happy to help author that post.

I might be conflating two things, but this seems like a conversation worth having, at least judging by us working together here to figure out the exact guidelines and what's appropriate.

The particulars of this video aside, I'm struggling to find the connection between promoting any individual or business who is volunteering, regardless of the upsells, from an .org perspective.

I can recognize that being able to promote oneself would entice more people to contribute. But if we are contributing from a place that has our own contributions as a piece of it, that doesn't feel like a contribution in the spirit of the project to me.

**Comment by jamiemarsland** [comment]
Just to reiterate "I'm 100% fine with having no call out whatsoever" - it will take me 10 mins to edit the video

**Comment by kaitohm** [comment]
@jamiemarsland 

> Can I double check one thing - it seems a call out to a commercial company is ok in the presenter bio from what I can see on https://learn.wordpress.org/ ?

That is correct. [Our current processes](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/#guidelines) allows this.

Also, thank you for offering to have no call out whatsoever.

---

I think we have a good discussion happening here, and we've identified some points that need to be taken to a public discussion. Here is my proposal for next steps. What do folks think?

### Rapid website recreation video

For simplicity's sake, and since @jamiemarsland offered, how about we publish this first one with no call-out at the end. Mentioning the speaker's company in the bio would follow current guidelines and be appropriate.

@jonathanbossenger Would you be able to work with Jamie to publish this first video? I think it would be appropriate to publish this as a Tutorial if we can add a few learning outcomes and comprehension questions. We should start a new GitHub issue to track this.

### The promotion of individuals/businesses in content on Learn

This is a conversation I'd like to invite the wider community (not even just Training) to weigh in on. And to do that, I'd like to move that discussion into a blog post on https://make.wordpress.org/training/

@nomad-skateboarding-dev Shall we coordinate in Slack to co-author this post? We can draft in Google Docs, share it with the Faculty to get an editorial review, and then publish it on the team blog.

**Comment by jamiemarsland** [comment]
Here's the edited video with the new ending :) https://vimeo.com/818643828

**Comment by nomad-skateboarding-dev** [comment]
> @nomad-skateboarding-dev Shall we coordinate in Slack to co-author this post? We can draft in Google Docs, share it with the Faculty to get an editorial review, and then publish it on the team blog.

This sounds great @bsanevans ! Reach out to me in slack today, or when you have space, and I'm in for collaborating. 🙂🙌

**Comment by kaitohm** [comment]
@jamiemarsland This looks good! Thank you for the edits.

I'll close this issue out now. Please open a new "Tutorial Template" issue [from this link](https://github.com/WordPress/Learn/issues/new/choose) so we can track the publication in GitHub. Please ping @jonathanbossenger in the new issue, so he is notified there as well 🙇
